docs/api/ListenerClose.md

```diff
@@ -23,7 +23,7 @@ A valid handle to an open listener object.
 
 # Remarks
 
-`ListenerClose` frees all allocated resources associated with the listener handle. If a listener has not had [ListenerStop](ListenerStop.md) called on it at the time `ListenerClose` is called, [ListenerStop](ListenerStop.md) is invoked internally. Due to this, a server application **MUST NOT** call `ListenerClose` within any callback unless it has already called [ListenerStop](ListenerStop.md) outside of any callbacks.
+`ListenerClose` frees all allocated resources associated with the listener handle. If a listener has not had [ListenerStop](ListenerStop.md) called on it at the time `ListenerClose` is called, [ListenerStop](ListenerStop.md) is invoked internally. `ListenerClose` may be called on any Listener callback.
```

Comment:
This needs to be changed that ListenerClose can only be called from another thread or the StopComplete event.